docs/hub/spaces-sdks-docker-panel.md

```diff
@@ -0,0 +1,51 @@
+# Panel on Spaces
+
+[Panel](https://panel.holoviz.org/) is an open-source Python library that lets you easily build powerful tools, dashboards and complex applications entirely in Python. It has a batteries-included philosophy, putting the PyData ecosystem, powerful data tables and much more at your fingertips. High-level reactive APIs and lower-level callback based APIs ensure you can quickly build exploratory applications, but you aren’t limited if you build complex, multi-page apps with rich interactivity. Panel is a member of the [HoloViz](https://holoviz.org/) ecosystem, your gateway into a connected ecosystem of data exploration tools.
+
+Visit [Panel documentation](https://panel.holoviz.org/) to learn more about making powerful applications. 
+
+## 🚀 Deploy Panel on Spaces
+
+You can deploy Panel on Spaces with just a few clicks:
+
+<a href="https://huggingface.co/new-space?template=Panel-Org/panel-template"> <img src="https://huggingface.co/datasets/huggingface/badges/raw/main/deploy-to-spaces-lg.svg"/> </a>
+
+There are a few key parameters you need to define: the Owner (either your personal account or an organization), a Space name, and Visibility. In case you intend to execute computationally intensive deep learning models, consider upgrading to a GPU to boost performance. 
+
+<img src="https://raw.githubusercontent.com/holoviz/panel/main/doc/_static/images/hugging_face_space.png" style="width:70%"></img>
+
+Once you have created the space, it will start out in “Building” status, which will change to “Running” once your space is ready to go. 
```

Comment:
```suggestion
Once you have created the Space, it will start out in “Building” status, which will change to “Running” once your Space is ready to go.
```